html/index.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
+<meta name="viewport" content="width=device-width,height=device-height,initial-scale=1.0"/>
+<title>Tic-Tac-Toe game</title>
+<link rel = "stylesheet" type = "text/css" href = "../css/main.css" />
+<script type="text/javascript">
+        var global_key = '';
+		var count = 0;
+		var s = 0;
+		var a = 0;
+		var c = 0;
+		var t = [];
+		var b = 0;
+		var d = "file:///C:/Projects/game/html/index.html";
+        var img = {};
+	    img["x"] ="../image/IMG_20170707_154212.jpg";
+	    img["o"] = "../image/IMG_20170707_153859.jpg";
+		 var keys = Object.keys(img);
+	     var keys1 = keys[Math.floor(Math.random()*keys.length)];
+		 function turn(){
+							 if(keys1 == keys[0])
+							 {
+							    document.getElementById("win").innerHTML = " Player X turn";
+								
+						     }else{
+							    document.getElementById("win").innerHTML = " Player 0 turn";
+							}
+			            }			
+		 //random image display
+        function displayRandomImage(ele){                      
+        if(global_key == ''){
+	              
+	               global_key = keys1;
+                   var img1 = img[keys1];
+		           if(ele.children[0].getAttribute('src') == ' '){
```

Comment:
Take ele.children[0].getAttribute('src') in single variable and use it  to check it's value wherever required